src/widgets/lights-puzzle.jsx

```diff
@@ -45,6 +43,7 @@ var RESET_BUTTON_STYLE = {
 
 var MAIN_TILE_SIZE = 50;
 
+/* eslint-disable no-unused-vars */
```

Comment:
Those were some unused functions that still looked useful so I didn't think I should delete them in case they were used for debugging or something?